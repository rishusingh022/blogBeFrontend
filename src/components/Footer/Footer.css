.footer {
  background-color: #121212;
  padding-top: 2rem;
  padding-bottom: 2rem;
  color: white;
}

@media screen and (max-width: 550px) {
  .footer {
    flex-direction: column;
    text-align: center;
    gap: 1rem;
  }
}
