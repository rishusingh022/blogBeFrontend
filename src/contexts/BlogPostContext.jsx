import React, { createContext, useState } from "react";

export const BlogPostContext = createContext({});

export const BlogPostProvider = ({ children }) => {
  const [posts, setPosts] = useState(null);

  return (
    <BlogPostContext.Provider value={{ posts, setPosts }}>
      {children}
    </BlogPostContext.Provider>
  );
};
