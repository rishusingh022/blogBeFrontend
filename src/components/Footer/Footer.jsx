import React from "react";
import "./Footer.css";
function Footer() {
  return <div className="body-padding footer">&copy; artifact.com 2019</div>;
}

export default Footer;
