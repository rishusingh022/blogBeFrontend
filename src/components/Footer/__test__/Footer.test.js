import { render } from "@testing-library/react";
import React from "react";
import { BlogPostProvider } from "../../../contexts/BlogPostContext";
import Footer from "../Footer";

describe("Footer", () => {
  it("should render corrently", () => {
    const view = render(
      <BlogPostProvider>
        <Footer />
      </BlogPostProvider>
    );
    expect(view.asFragment()).toMatchSnapshot();
  });
});
