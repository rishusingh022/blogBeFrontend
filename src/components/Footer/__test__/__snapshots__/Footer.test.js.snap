// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render corrently 1`] = `
<DocumentFragment>
  <div
    class="body-padding footer"
  >
    Â© artifact.com 2019
  </div>
</DocumentFragment>
`;
